t/plugin/elasticsearch-logger.t

```diff
@@ -551,3 +551,32 @@ PTQvJEaPcNOXcOHeErC0XQ==
     }
 --- response_body
 passed
+
+
+
+=== TEST 14: to show that different endpoints will be chosen randomly
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            for i = 1, 10 do
+                local code, body = t('/hello', ngx.HTTP_GET)
+                ngx.say("code: ", code)
+            end
+
+        }
+    }
+--- response_body
+code: 200
+code: 200
+code: 200
```

Comment:
We can count the code like https://github.com/apache/apisix/blob/22be6d08cef7de16c3f8a303e75c1872378c3dee/t/node/healthcheck2.t#L139

Reply:
Done.